lib/galaxy/util/path/__init__.py

```diff
@@ -82,7 +89,7 @@ def safe_relpath(path):
     return not (isabs(path) or normpath(path).startswith(pardir))
 
 
-def unsafe_walk(path, whitelist=None):
+def unsafe_walk(path, whitelist=None, username=None):
```

Comment:
@natefoo afaik you spent a lot of time on these recently so you might want to have a look

Reply:
@martenson thanks for #5350 so I don't lose track of this.